ganga/GangaCore/GPIDev/Adapters/IVirtualization.py

```diff
@@ -0,0 +1,35 @@
+##########################################################################
+# Ganga Project. http://cern.ch/ganga
+#
+##########################################################################
+from GangaCore.GPIDev.Schema import Schema, Version, SimpleItem
+from GangaCore.GPIDev.Base import GangaObject
+from GangaCore.GPIDev.Lib.File.File import File
+
+class IVirtualization(GangaObject):
+
+    """
+    Contains the interface for all virtualization classes, all virtualization classes should inherit from this object.
+    """
+
+    def __init__(self, image):
+        super(IVirtualization, self).__init__()
+        self.image = image
+
+    _schema = Schema(Version(1, 0), {
+        'image': SimpleItem(defvalue="", typelist=[str, File], doc='Link to the container image')
```

Comment:
I was unclear in my specification here. The object that we want to accept is a `GangaFile` object. See https://github.com/ganga-devs/ganga/blob/develop/ganga/GangaCore/GPIDev/Adapters/IGangaFile.py. This will allow us to attach files from the Local file system but also a file that sits on a Google Drive as an example.

Reply:
Was this addressed yet? I don't see the changes on this branch in Github. I think that other than this the branch looks ready to merge.

Reply:
@alexanderrichards sorry, Still not updated. I'm looking into it. But I'm having trouble implementing it.

Reply:
I implemented this today.